js/components/discussions/thread.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+<div>
+    <div class="list-group-item" :id="discussionIdAttr" @click="toggleDiscussions">
+        <div class="format-label pull-left">
+            <avatar :user="question.posted_by"></avatar>
+        </div>
+        <span class="list-group-item-link">
+            <a href="#{{ discussionIdAttr }}"><span class="fa fa-link"></span></a>
+        </span>
+        <h4 class="list-group-item-heading ellipsis open-discussion-thread">
+            <span>{{ discussion.title }}</span>
+        </h4>
+        <p class="list-group-item-text ellipsis open-discussion-thread list-group-message-number-{{ discussion.id }}">
+            {{ _('Discussion started on {created} with {count} messages.',
+                 {created: discussion.created, count: responses.length})
```

Comment:
Never used filters like that, it was a proposal. Maybe filters are not usable in this case but you have 2 others possibilities:
- use a computed property
- use a method